memgpt/memory.py

```diff
@@ -338,6 +338,16 @@ def insert(self, memory_string):
             # breakup string into passages
             for node in parser.get_nodes_from_documents([Document(text=memory_string)]):
                 embedding = self.embed_model.get_text_embedding(node.text)
+                # fixing weird bug where type returned isn't a list, but instead is an object
+                # eg: embedding={'object': 'list', 'data': [{'object': 'embedding', 'embedding': [-0.0071973633, -0.07893023,
+                if isinstance(embedding, dict):
+                    try:
+                        embedding = embedding["data"][0]["embedding"]

```

Comment:
@sarahwooders I'm not sure if this is a llama-index bug, or if it's to do with how we're returning the payload from the hosted embedding server